doc/in-toto.md

```diff
@@ -23,3 +23,4 @@ A framework to secure the integrity of software supply chains https://in-toto.io
 * [in-toto sign](in-toto_sign.md)	 - Provides command line interface to sign in-toto link or layout metadata
 * [in-toto verify](in-toto_verify.md)	 - Verify that the software supply chain of the delivered product
 
+###### Auto generated by spf13/cobra on 18-Oct-2021
```

Comment:
Can you remove these comments from the PR via setting  DisableAutoGenTag: true, in the cobra RootCmd again?

Reply:
Added DisableAutoGenTag: true to remove comments.